ods_ci/tests/Resources/CLI/ModelRegistry/updateIPYNB.py

```diff
@@ -0,0 +1,44 @@
+import json
```

Comment:
I'm not 100% sure about what the goal of this script is, but it seems like you're fetching some values with `oc` and then substituting them in the notebook file. 
I don't think this is the best way to do this to be honest - if you need to fetch values at runtime from your notebook, wouldn't it be better to run the oc commands from the notebook itself?
If these values are not taken/created at runtime, wouldn't it be better to have the notebook with the values available already?

Reply:
The python client requires the Domain of the hosting cluster, so it's not possible to have the notebook file with pre-done values. That's the main reason I'm keeping it locally, so I can work with it and then upload directly to Jupyter Notebook.

Reply:
I had tried to get the values directly in the notebook file at run time, but the permissions are not available to retrieve the Domain.

Reply:
I'm not 100% sure what you mea by Domain, but would this work?
```
import os
test = os.getenv("NOTEBOOK_ARGS")
dashboard = test.split('"hub_host":')[1].split(",")[0].strip('"')
console = url.replace("rhods-dashboard-redhat-ods-applications","console-openshift-console")
print(f'Env variable: \n{test}')
print('--------------------------')
print('Dashboard url: '+url)
print('--------------------------')
print('Console url: '+console)
```

Reply:
As I have to get the domain for the setup to be used elsewhere I think it makes more sense to use during setup for both cert generation and the notebook file.  I think this way we will have less code.  For now I'm under a bit of time pressure to get this merged.  I'm really interested in this though, _test = os.getenv("NOTEBOOK_ARGS")_. I wasn't aware this was available.

Reply:
What I mean by Domain is the part of the route urls that is static and does not change. For example apps.tony-modelreg.osp.rh-ods.com

Reply:
>For now I'm under a bit of time pressure to get this merged

You can definitely get the domain from that environment variable, please refactor this when you update the test